ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Selection.cs

```diff
@@ -285,6 +286,10 @@ private List<Element> GetSelectionFilterObjects(ISpeckleConverter converter, ISe
       }
       catch (Exception ex)
       {
+        if (ex is ModificationForbiddenException)
+          throw new SpeckleException($"Modification Forbidden: please open another View and try again.", ex);
+
+
```

Comment:
Only comment I would have here is to catch this in a separate catch clause above this one.

But not a real blocker

Reply:
Makes sense, txs! 🤦‍♂️